controller/controller_server/csi_controller_server.py

```diff
@@ -350,57 +291,31 @@ def ControllerUnpublishVolume(self, request, context):
 
             logger.info("finished ControllerUnpublishVolume")
             return csi_pb2.ControllerUnpublishVolumeResponse()
-
+        except ObjectIdError as ex:
+            return handle_exception(ex, context, grpc.StatusCode.INVALID_ARGUMENT,
+                                    array_errors.VolumeAlreadyUnmappedError)
```

Comment:
an array error is not a response type

Reply:
@oriyarde Do you want me to change it?

Reply:
handled in https://github.com/IBM/ibm-block-csi-driver/pull/496